### PR TITLE
Fix machine check warning status

### PR DESCRIPTION
**Pull request by benwaffle**
Per corrosion:
```
./state.db> select distinct status from consul_checks;
+----------+
| status   |
+----------+
| passing  |
| critical |
| warning  |
+----------+
```

### CODE REVIEW

**Review by kzys — COMMENTED** [review]
The time to defile constants?

**Comment by benwaffle** [comment]
Maybe, but I can't see a clean way to make the JSON unmarshaller validate the string enum

**Comment by benwaffle** [comment]
@kzys something like that?

File: internal/format/format.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func passingChecks(checks []api.CheckState) (n int) {
 
 func warnChecks(checks []api.CheckState) (n int) {
 	for _, check := range checks {
-		if check.Status == "warn" {
+		if check.Status == "warning" {
```
Comment (kzys) [inline internal/format/format.go:127]: Here too?

**Review by kzys — APPROVED** [review]

File: internal/format/format.go (hunk 1 above)
Comment (benwaffle) [reply internal/format/format.go:127]: that's a different field in types.go. I only changed machine_types.go

**Pull request merged**